### ISSUE TITLE
ovspy not installed?

### ISSUE THREAD

**Issue by fvalle1** [Elemento-Modular-Cloud/helpcenter#45]
```
[morro@localhost ~]$ journalctl -fu networkserver.service
Dec 10 12:51:02 localhost.localdomain python3[46259]:   File "/opt/elemento/elemento-monorepo-server/elemento-network-server/networkserver.py", line 46, in <module>
Dec 10 12:51:02 localhost.localdomain python3[46259]:     import ovspy.client
Dec 10 12:51:02 localhost.localdomain python3[46259]: ModuleNotFoundError: No module named 'ovspy'
Dec 10 12:51:02 localhost.localdomain systemd[1]: networkserver.service: Main process exited, code=exited, status=1/FAILURE
Dec 10 12:51:02 localhost.localdomain systemd[1]: networkserver.service: Failed with result 'exit-code'.
```

**Comment by fvalle1** [comment]
@filippo-ferrando `sudo /opt/elemento/venv/bin/python3 -m pip install ovspy` ovspy is installed in atomos-rpm
@morrolinux can give further details

**Comment by filippo-ferrando** [comment]
the alma installer script doesn't actually install ovspy via pip, it installs `openvswitch3.1.x86_64 openvswitch3.1-ipsec.x86_64 ` if is not the correct behavior i can fix it

**Comment by filippo-ferrando** [comment]
fixed on [0a71817d](https://github.com/Elemento-Modular-Cloud/atomos-rpm/commit/0a71817d8520076c769a3138611684228d8fcae0)
